src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.Core.cs

```diff
@@ -45,6 +45,10 @@ internal ResourceReader(Stream stream, Dictionary<string, ResourceLocator> resCa
             ReadResources();
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL3050:RequiresDynamicCode",
```

Comment:
Similar here. Why is the trim warning not enough?

Reply:
`InitializeBinaryFormatter `uses [MakeGenericType ](https://github.com/dotnet/runtime/pull/62562/files)which has the `RequiresDynamicCode `annotations